src/commands/Moderation/multiunban.ts

```diff
@@ -86,7 +90,7 @@ export default class extends Command {
 				reason,
 			);
 
-			fullList.push(`**${target.tag}** \`\`(${target.id})\`\` \`\`[#${infID}]\`\``);
+			fullList.push(`**${target.tag}** \`(${target.id})\` \`[#${infID}]\``);
 		}
 
 		if (validTargets.length)
```

Comment:
This seems to not be working consistently. Could probably use `fullList.length` instead. Would also then be able to remove the array splice calls.